### ISSUE TITLE
JBang Project Wizard

### ISSUE THREAD

**Issue by linux-china** [jbangdev/jbang-idea#18]
Create JBang app from Project Wizard. 

Features:

* Create JBang project from user defined template
* Setup IDEA project automatically

https://plugins.jetbrains.com/docs/intellij/project-wizard.html

**Comment by maxandersen** [comment]
This would use the same as used in create script but use it to create a project not just a file?

**Comment by linux-china** [comment]
Yes, create a new project from JBang template.

**Comment by linux-china** [comment]
Reuse JavaModuleBuilder, and developers can choose Groovy, Kotlin languages, then the plugin create script file according developers' choice, for example Hello.java, Hello.groovy or Hello.kt

<img width="767" alt="Screen Shot 2022-01-17 at 5 08 24 PM" src="https://user-images.githubusercontent.com/46711/149853579-e4105ddd-3623-4689-9096-fb54a07cb919.png">

**Comment by maxandersen** [comment]
Shouldn't it show/allow jbang templates ?

**Comment by linux-china** [comment]
I will implement this feature later because of custom UI builder.

**Comment by nandorholozsnyak** [comment]
> Reuse JavaModuleBuilder, and developers can choose Groovy, Kotlin languages, then the plugin create script file according developers' choice, for example Hello.java, Hello.groovy or Hello.kt
> 
> <img alt="Screen Shot 2022-01-17 at 5 08 24 PM" width="767" src="https://user-images.githubusercontent.com/46711/149853579-e4105ddd-3623-4689-9096-fb54a07cb919.png">

I know the plugin has a cool create new JBang script window, but will the template selector implemented in the Module creation wizard too?
https://github.com/jbangdev/jbang-idea/issues/43

**Comment by linux-china** [comment]
@nandorholozsnyak  IntelliJ IDEA 2022.1 has different UI and I will take a look and make a good design.

![image](https://user-images.githubusercontent.com/46711/158228965-426038d9-d3f8-4ccd-bbae-a796dc2573cb.png)

**Comment by fbricon** [comment]
I installed the plugin on IJ 2021.3, and saw I could create a JBang project, but after updating to 2022.1.2, I no longer see the possibility of creating a JBang project.

**Comment by linux-china** [comment]
@fbricon  Yes, IntelliJ IDEA project wizard UI has been changed, and I will use new API for wizard UI.

**Comment by maxandersen** [comment]
closing as no progress afaik and also don't really think jbang scripts/apps fits into intellij project wizard as is atm.
